internal/service/pushbasedlogexport/resource_test.go

```diff
@@ -196,7 +196,7 @@ resource "aws_iam_role_policy" "test_policy" {
     "Version": "2012-10-17",
     "Statement": [
       {
-        "Effect": "Allow",
+        "Effect": "Deny",
```

Comment:
This would need S3 permissions because Atlas creates a test object in the bucket for verification. That's why right now the test fails with BUCKET_VERIFICATION_FAILED error

Reply:
yup, thanks for flagging that - I am fixing it